src/index.d.ts

```diff
@@ -368,6 +368,12 @@ declare namespace RamdaAdjunct {
          */
         list(...items: any[]): any[];
 
+        /**
```

Comment:
I'd prefer the following as description. It's basically a compilate of `when` and `of` descriptions.
```
Returns a singleton array containing the value provided.
If value is already an array, it is returned as is.